iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Injection.java

```diff
@@ -18,4 +21,27 @@
      */
     Terminal getTerminal();
 
+    default List<OperationalLimits> getOperationalLimits() {
+        return Collections.emptyList();
+    }
+
+    default <L extends OperationalLimits> L getOperationalLimits(LimitType type, Class<L> limitClazz) {
+        return null;
+    }
+
+    default CurrentLimitsAdder newCurrentLimits() {
+        throw new UnsupportedOperationException();
+    }
+
+    default ApparentPowerLimitsAdder newApparentPowerLimits() {
+        throw new UnsupportedOperationException();
+    }
+
+    default ActivePowerLimitsAdder newActivePowerLimits() {
+        throw new UnsupportedOperationException();
+    }
+
+    default VoltageLimitsAdder newVoltageLimits() {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
Some of the operation limits are weird for injections. In IIDM, we do not have sufficient information to compute them.

Reply:
Same answer as above